src/hypermedia-constants.js

```diff
@@ -665,6 +665,7 @@ export const Actions = {
 		},
 		activityUsageCollection: {
 			setCollectionPaging: 'set-collection-paging',
+			startAddNewActivity: 'start-add-new-activity',
```

Comment:
There is also `d2l-hypermedia-constants` lib. So, I am not sure which one is the main repo. Maybe we need to update both which I hope we standardize it to just 1.